homeassistant/components/balboa/select.py

```diff
@@ -0,0 +1,52 @@
+"""Support for Spa Client selects."""
+
+from pybalboa import SpaClient, SpaControl
+from pybalboa.enums import LowHighRange
+
+from homeassistant.components.select import SelectEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .entity import BalboaEntity
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Set up the spa select entity."""
+    spa: SpaClient = hass.data[DOMAIN][entry.entry_id]
+    async_add_entities([BalboaTempRangeSelectEntity(spa.temperature_range)])
+
+
+class BalboaTempRangeSelectEntity(BalboaEntity, SelectEntity):
+    """Representation of a Temperature Range select."""
+
+    _attr_icon = "mdi:thermometer-lines"
+
+    def __init__(self, control: SpaControl) -> None:
+        """Initialise the switch."""
+        super().__init__(control.client, "TempHiLow")
+        self._attr_name = "Temperature range"
+        self._attr_unique_id = "temperature_range"
+        self._attr_translation_key = "temperature_range"
+        self._attr_options = [
+            LowHighRange.LOW.name.lower(),
+            LowHighRange.HIGH.name.lower(),
+        ]
```

Comment:
Set this as class attributes (outside of `__init__`) instead of instance attributes